app/javascript/components/MainContainer/AddRestaurantForm/AddRestaurantForm.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import './add_restaurant_form.css'
+
+class AddRestaurantForm extends Component {
+    constructor(props){
+        super(props);
+    }
+
+    render () {
+        const { handleSubmit, createRestaurant, cuisines, cancel, errorMsg} = this.props;
+        cuisines[0] = "";
```

Comment:
It's generally not a good idea to modify the props.
It's also generally not a good idea to have any kind of side-effect actions in `render` (it should remain a pure function, only returning something with no side-effects).